source/testnet/references/concordium-client.rst

```diff
@@ -45,7 +30,7 @@ directory:
 **This page will describe the commands that are related with configuration of
 the client, but the rest of available commands will be discussed on the pages
 where the features that use them are documented. Some commands will perform**\
-`queries`_\ **and others send** :ref:`transactions<transactions>`\ **.**
+:ref:`queries<testnet-query-node>`\ **and others send** :ref:`transactions<transactions>`\ **.**
```

Comment:
Hele den her paragraf er i bold. Ved ikke om det er meningen, eller om det er en fejl der er sket nu

Reply:
Der mangler også et par mellemrum rundt om query linket

Reply:
Could we please stick to English.

Reply:
Ooops, yes ofcourse. That was a brainfart.

Reply:
I think it would fit better as a `..note::` rather than bold